### DISCUSSION TITLE
Bug: (the bug you found)The WAV file is too large, how can I reduce it? 1.2 GB for video and 1.2 GB for transcribed audio files. How to set it smaller

### DISCUSSION THREAD

**Discussion by wzgrx** [CarlGao4/Demucs-Gui#68] [Q&A] [answered]
### happen

The WAV file is too large, how can I reduce it? 1.2 GB for video and 1.2 GB for transcribed audio files. How to set it smaller

### What is the platform you are using?

Windows

**Comment by CarlGao4** [comment]
What is your save options?

**Reply by wzgrx** [reply]
Default settings, is there an adjustment option

**Reply by CarlGao4** [reply, marked as answer]
Under save options, you can change it into FLAC int16
And if you change the encoder to FFMpeg, you can even save MP3 and AAC but it IS NOT LOSSLESS

**Comment by wzgrx** [comment]
> What is your save options?



> Under save options, you can change it into FLAC int16 And if you change the encoder to FFMpeg, you can even save MP3 and AAC but it IS NOT LOSSLESS

ok thx
